lightningd/chaintopology.c

```diff
@@ -341,6 +342,8 @@ static void updates_complete(struct chain_topology *topo)
 	if (topo->tip != topo->prev_tip) {
 		/* Tell lightningd about new block. */
 		notify_new_block(topo->bitcoind->ld, topo->tip->height);
+		channel_notify_new_block(topo->bitcoind->ld,
+					 topo->tip->height);
```

Comment:
Minor cleanup:

I'd put the call to notify_new_block() in lightningd.c, rename the current one to (say) htlcs_notify_new_block() and have notify_new_block() call htlcs_notify_new_block and channel_notify_new_block.

it's a bit cleaner to have it outside chaintopology I think...